README.md

```diff
@@ -1,2 +1,44 @@
 # terraform-azure-sensor
+
 Terraform for Corelight's Azure Cloud Sensor Deployment.
+
+<img src="docs/overview.svg" alt="overview">
+
+## Usage
```

Comment:
Do we not want to follow the GCP sensor pattern where we link off to the example?

Reply:
Currently this Readme deviates from our others, style wise.

Reply:
I don't think it hurts to have a usage section and examples. 🤷